lib/context.ml

```diff
@@ -34,6 +34,14 @@ let get_secrets_exn ctx =
   | None -> context_error "secrets is uninitialized"
   | Some secrets -> secrets
 
+let get_config_filename ctx repo_url =
+  match get_secrets_exn ctx with
+  | exception Context_error _ -> ctx.config_filename
```

Comment:
I think that it would be better without this catch, but the tests are failing if I omit it.